src/api/lib/tasks/statistics/current.rake

```diff
@@ -0,0 +1,24 @@
+namespace :statistics do
+  desc 'Exports the current number of users, projects, packages, etc. to a file'
+  task current_numbers: :environment do
+    # The context of the file is NOT prepared to be used by R, just a easy way to get useful information
+    elements = HistoryElement::Request
+               .where(type: "HistoryElement::RequestAccepted")
+               .joins("INNER JOIN bs_requests ON bs_requests.id = history_elements.op_object_id")
+               .pluck('bs_requests.created_at', 'history_elements.created_at')
+    time = 0 # time in seconds
+    elements.sum { |element| time += (element[1].to_i - element[0].to_i) }
```

Comment:
@Ana06 @mdeniz The point of using sum was to not have to assign the value in a loop

```ruby time = elements.sum { |element| element[1].to_i - element[0].to_i } ```

or even (didn't test though)

```ruby time = elements.sum { |request_created, history_created| history_created.to_i - request_created.to_i } ```